### PR TITLE
test(bridge): Fix flaky UI test on first day of a month

### PR DESCRIPTION
**Pull request by Kirdock**
On the first day of the month, some UI tests were not successful because it was not able to select two different days

Signed-off-by: Klaus Strießnig <k.striessnig@gmail.com>

### CODE REVIEW

**Review by renepanzar — APPROVED** [review]
LGTM!

File: bridge/cypress/support/pageobjects/TriggerSequenceSubPage.ts [hunk 1]
```diff
@@ -85,7 +85,16 @@ export class TriggerSequenceSubPage {
...
+  public selectDateTime(
+    calElement: number,
+    hours: string,
+    minutes: string,
+    seconds: string,
+    goBackMonths: number
+  ): this {
+    for (let i = 0; i < goBackMonths; ++i) {
+      cy.get('.dt-calendar-header-button-prev-month').click();
+    }
```
Comment (heinzburgstaller) [inline bridge/cypress/support/pageobjects/TriggerSequenceSubPage.ts:97]: Suggestion: You could do
```
[...Array(goBackMonths).keys()].forEach(() => cy.get('.dt-calendar-header-button-prev-month').click())
```
To get rid of for loop

Comment (Kirdock) [reply bridge/cypress/support/pageobjects/TriggerSequenceSubPage.ts:97]: This is still a for-loop, even a more complicated one.
You create an array with the length of 5, get the keys via iterator, create another array out of it and then iterate over it.
Also `array.forEach()` is as the same as `for (const i of array)`

Comment (Kirdock) [reply bridge/cypress/support/pageobjects/TriggerSequenceSubPage.ts:97]: I completely disagree here and for me, the for-loop approach is much more read-friendly than the forEach

**Pull request merged**